tests/SprykerEcoTest/Zed/Braintree/Business/Order/SaverTest.php

```diff
@@ -274,7 +319,27 @@ protected function createAndGetCustomerEntity()
      */
     protected function createAndGetOrderEntity(SpySalesOrderAddress $billingAddressEntity, SpyCustomer $customerEntity)
     {
-        $orderEntity = (new SpySalesOrder())
+        $order = (new SpySalesOrderQuery())->filterByIdSalesOrder(1)->findOne();
```

Comment:
this is strange
you should not rely on ID 1 here

Reply:
Fixed.